### PR TITLE
Seedingwork

### PR DESCRIPTION
**Pull request by rikotacards**
Newly added: 
database/psql.seed.js, used for creating and writing data generated by faker into cvs file, starting from line 33.
psql.schema.sql (root directory), used to create tables, then copy the csv files into postgres db

### CODE REVIEW

File: database/psql.seed.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+const generateBooks = () =>{
+  
+  var books = {
+    title: data.title(),
+    description: data.description(),
+    author_id: data.author_id({min:1, max:1000000}),
+    published_year: data.year({ min: 1920, max: 2019}),
+    cover: data.cover + data.author_id({min:1,max: 7})+'.jpg',
+    status: data.status}
```
Comment (bkwon94) [inline database/psql.seed.js:15]: Looks good, but just move the curly brace down!

File: database/dummy_data.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const description = faker.lorem.paragraphs;
 const year = faker.random.number;
 const author_id = faker.random.number;
```
Comment (bkwon94) [inline database/dummy_data.js:8]: What is the purpose of this file?

File: database/psql.seed.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    return books
+}
+
+const generateAuthor = () => {
```
Comment (bkwon94) [inline database/psql.seed.js:20]: Is your data generation and seeding in the same file?

File: database/psql.seed.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+writableStream.on("finish", function(){
+  console.log("Books DONE!");
+});
+
+csvStream.pipe(writableStream);
+csvStream.on('drain', function() {
+  writeBooks();
+});
+var i =10000000;
```
Comment (bkwon94) [inline database/psql.seed.js:44]: Maybe use count instead of i for better naming.

File: database/psql.seed.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+function writeAuthors(){
+
+while( numOfAuthors>0){
+if(!csvStreamAuthors.write(generateAuthor())){
+  numOfAuthors--
+  return
+  }
+numOfAuthors--
+}
+var endAuthors = new Date() - startAuthors
```
Comment (bkwon94) [inline database/psql.seed.js:87]: You could use console.time and console.timeEnd to time your code. You can have multiple at once

**Review by bkwon94 — APPROVED** [review]

File: database/psql.seed.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+csvStream.pipe(writableStream);
+csvStream.on('drain', function() {
+  writeBooks();
+});
+var i =10000000;
+
+function writeBooks(){
+
+while( i>0){
```
Comment (anacollado) [inline database/psql.seed.js:48]: Maybe clean up the formatting here - looks you're missing some spaces and semi-colons in the code (lines 48-54)

**Pull request merged**